_includes/_favicon.html

```diff
@@ -4,27 +4,27 @@
 *   
 {% endcomment %}
 {% comment %} Regular Old-Skool Favicon: {% endcomment %}
-	{% if site.favicon-32x32 %}<link rel="icon" sizes="32x32" href="{{ site.url }}/assets/img/{{ site.favicon-32x32 }}">{% endif %}
+	{% if site.favicon-32x32 %}<link rel="icon" sizes="32x32" href="assets/img/{{ site.favicon-32x32 }}">{% endif %}
```

Comment:
If you use the 'local' target in the Makefile, i.e. `% make local`, then `_config_dev.yml` overwrites the setting of `site.url` and you will get images served from the local filesystem.  

I would say that the existing approach, with absolute URLs, isn't broken, so I'd prefer not to change it.